Assets/HoloToolkit/BuildAndDeploy/Editor/XdeGuestLocator.cs

```diff
@@ -54,7 +54,13 @@ public static void FindGuestAddressAsync()
                 HasData = false;
                 GuestIpAddress = IPAddress.None;
 
-                var internalSwitchAddressInfo = GetInternalSwitchAddressInfo();
+                var internalSwitchAddressInfo = null;
```

Comment:
Also an implicitly typed local variable declaration cannot be initialized with `null'